testing/scenario_app/README.md

```diff
@@ -21,10 +21,14 @@ platform channel.
 
 ## Running for iOS
 
+Build the `ios_debug_sim_unopt` engine variant, and run
+
 ```sh
-./build_and_run_ios_tests.sh
+./run_ios_tests.sh
 ```
 
+in your shell.
```

Comment:
While you are here, do you mind adding a little instruction for running locally through xcode?
Something like:
```
To run or debug in XCode, open the proj file located in  <path_to_engine>/src/out/ios_debug_sim_unopt/scenario_app/Scenarios/Scenarios.xcodeproj, and run all tests with cmd+u or use XCode UI to run selected tests.
```

Reply:
Will do

Reply:
thanks!